### DISCUSSION TITLE
AppCSXCAD missing libjawt.so

### DISCUSSION THREAD

**Discussion by frederickbrown** [thliebig/openEMS-Project#116] [General]
I installed the openEMS package on a Debian 12 system using apt.  When I issue "AppCSXCAD" on the command line, I get this:

AppCSXCAD: error while loading shared libraries: libjawt.so: cannot open shared object file: No such file or directory

Any idea what version of libjawt.so AppCSXCAD is looking for, or where is is looking for it?

**Comment by thliebig** [comment]
I never heard of that lib, not sure which other lib may have this dependency... Did you try to build from source? 
Not sure how old the apt package is anyhow...
